dash.js

```diff
@@ -743,6 +744,34 @@ var MyDash = class DashToDock_MyDash {
             }
         }
 
+        if (this._dtdSettings.get_boolean('show-mounts')) {
+            if (!this._removables) {
+                this._removables = new Locations.Removables();
+                this._signalsHandler.addWithLabel('show-mounts',
+                    [ this._removables,
+                      'changed',
+                      this._queueRedisplay.bind(this) ]);
+            }
+            Array.prototype.push.apply(newApps, this._removables.getApps());
```

Comment:
Why not just calling `newApps.push(this._removables.getApps())` ?

Reply:
At the time I wrote the original patch, the js version in ghome-shell didn't directly support calling push and this was the way you had to do it. I'm not surprised that it's directly callable now, but I wouldn't change it until it was clear what the minimum supported gnome-shell version is.